File: docs/tutorial/magics/symbol/simple_symbol.py
# (C) Copyright 1996-2016 ECMWF.
# 
# This software is licensed under the terms of the Apache Licence Version 2.0
# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0. 
# In applying this licence, ECMWF does not waive the privileges and immunities 
# granted to it by virtue of its status as an intergovernmental organisation nor
# does it submit to any jurisdiction.


#importing Magics module
from magmacro import *

#Setting of the output file name
output = output({"output_formats":['ps', 'png'], 
		'output_name': 'simple_symbol'})


#Background Coastlines 
background = mcoast( {???})

#Import the airep data 
airep =  mgeo({ "geo_input_file_name" : "../airep.geo"})

#Define the simple contouring for msl
airep_symbol = msymb( {"symbol_type" : ???,
		    "symbol_colour" : ???,
			"symbol_height" : 0.1,
			"symbol_marker": 15 })

lines =["My first symbol plotting"]

title = mtext({???})


#To the plot
plot(???)

File: docs/tutorial/magics/symbol/simple_symbol.f
C (C) Copyright 1996-2016 ECMWF.
C 
C This software is licensed under the terms of the Apache Licence Version 2.0
C which can be obtained at http://www.apache.org/licenses/LICENSE-2.0. 
C In applying this licence, ECMWF does not waive the privileges and immunities 
C granted to it by virtue of its status as an intergovernmental organisation nor
C does it submit to any jurisdiction.
C

      program  magics

C Define the title
      character*25 var
      dimension var(1)
      data var / +    'My first symbol plotting' /

C Open Magics
      call popen

C set the outut information
      call psetc(???)

C Define the background coastlines
      call pset?(???)
      call pcoast

C Load the data 
      call psetc('geo_input_file_name', '../airep.geo')
      call pgeo

c Define the attributes of the symbol plotting
      call psetc('symbol_type', ???)
      call pseti('symbol_marker', ???)
      call psetc('symbol_colour', ???)
      call psetr('symbol_height', ???)
      call psymb

C Add a text
      call pset?(???)
      call ptext


C Close magics : do the plot
      call ???

      end
